drivers/backup/backup.go

```diff
@@ -309,6 +309,12 @@ type ScheduleBackup interface {
 	// This wait function is for the backupschedule deletion with delete-backup option set.
 	WaitForBackupScheduleDeletion(ctx context.Context, backupScheduleName, namespace, orgID string,
 		clusterObj *api.ClusterObject, timeout time.Duration, timeBeforeRetry time.Duration) error
+
+	// GetScheduleBackupName gets the uid for the given schedule policy in an organization
```

Comment:
It is returning the schedule backup name right?
I would suggest to change the comment to something like this

```
// GetScheduleBackupName gets the name of latest Scheduled backup for the given backup schedule
```

Organization has no relevance here.

Reply:
I am considering renaming backupScheduleName to simply scheduleName, as the latter is more consistent with the UI.

Reply:
ok